[enterprise-4.16] OCPBUGS-41915 lack of explanation regarding numaresources-operator for disconnected environments (#84817)
openshift/openshift-docs: modules/cnf-deploying-the-numa-aware-scheduler.adoc

[id="cnf-deploying-the-numa-aware-scheduler_{context}"]
= Deploying the NUMA-aware secondary pod scheduler

> After you install the NUMA Resources Operator, follow this procedure to deploy the NUMA-aware secondary pod scheduler. 

Reviewer: Ideally, avoid self-referential language. 

https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

```suggestion
After you install the NUMA Resources Operator, deploy the NUMA-aware secondary pod scheduler [to do something].
```

metadata:
  name: numaresourcesscheduler
spec:

>   imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}" <1>

Reviewer: ```suggestion
  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}" # <1>
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml
